### PR TITLE
arm: Adios armemu

### PR DESCRIPTION
**Pull request by lioncash**

### CODE REVIEW

**Comment by yuriks** [comment]
:cake: :tada: :balloon:

**Comment by Alegend45** [comment]
I agree 100% with this change.

File: src/core/arm/interpreter/armcopro.cpp [hunk 1]
```diff
@@ -19,213 +19,45 @@
...
-//chy -------
-//#include "iwmmxt.h"
+// Dummy Co-processors.
 
-/* Dummy Co-processors.  */
-
-static unsigned
-NoCoPro3R(ARMul_State * state,
-unsigned a, ARMword b)
+static unsigned int NoCoPro3R(ARMul_State* state, unsigned int a, ARMword b)
```
Comment (yuriks) [inline src/core/arm/interpreter/armcopro.cpp]: You could change the brace style here to conform to the rest of the code, since you're touching it anyway.

**Comment by yuriks** [comment]
LGTM

**Comment by Subv** [comment]
LGTM 360 noscope instamerge

**Comment by bunnei** [comment]
LGTM

**Pull request merged**